django/thunderstore/repository/migrations/0029_create_namespaces_for_teams.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.1.7 on 2021-11-04 20:32
+
+from django.db import migrations
+
+
+def forwards(apps, schema_editor):
+    Namespace = apps.get_model("repository", "Namespace")
+    Team = apps.get_model("repository", "Team")
+    teams = Team.objects.all().order_by("id")
+    for team in teams:
+        Namespace.objects.create(team=team, name=team.name)
```

Comment:
Use a bulk create and create all namespaces with a single query instead of doing this. Better yet, use bulk inserts in batches of 2000 and use `.iterator()` to collect those batches. Batching is unnecessary given that we only have a few thousand teams, so just running a bulk create with everything in a single query should suffice.

Reply:
Doned